### ISSUE TITLE
[a11y] Colour scheme is not usable by people with red/green colour deficiency

### ISSUE THREAD

**Issue by LakeQueen** [cwackerfuss/react-wordle#586]
**Describe the bug**
The teal used to mark present letters, and the grey for absent letters, look indistinguishable to somebody with a red/green colour deficiency (protanopia, deuteranopia).

**Expected behavior**
Colours should either be a11y-friendly, or there should be an option to choose alternative palettes that are tailored to a specific colour deficiency.

**Desktop (please complete the following information):**
 - Windows 11, Chrome 108.0.5359.125

**Comment by igitur** [comment]
There is an option in the settings modal to switch to a high contrast colour scheme.
